Gordon360/Controllers/RecIM/MatchesController.cs

```diff
@@ -53,14 +53,29 @@ public ActionResult<MatchViewModel> GetMatchByID(int matchID)
         /// <summary>
         /// Updates Match Scores, Sportsmanship Ratings, and Team Status
         /// </summary>
+        /// <param name="matchID"></param>
         /// <param name="updatedMatch"></param>
         /// <returns></returns>
         [HttpPatch]
-        [Route("")]
-        public async Task<ActionResult> UpdateStats(MatchTeamPatchViewModel updatedMatch)
+        [Route("{matchID}/stats")]
+        public async Task<ActionResult> UpdateStats(int matchID, MatchStatsPatchViewModel updatedMatch)
+        {
+            var stats = await _matchService.UpdateTeamStats(matchID,  updatedMatch);
+            return CreatedAtAction("UpdateStats", stats);
+        }
+
+        /// <summary>
+        /// Updates Match STimes and Surfaces
```

Comment:
Not sure I understand what "STimes" is here. Start time, perhaps?

Reply:
I think I might've accidentally typed S but I think start times would make it more clear